src/custom-element-utilities.ts

```diff
@@ -77,20 +78,26 @@ export const createWebComponentClassFromBehavior = (
         behaviorInstruction
       );
 
-      const childContainer = createElementContainer(
-        container,
-        this,
-        targetInstruction,
-        children,
-        behaviorInstruction.partReplacements,
-        viewResources
-      );
-
-      const controller = behavior.create(childContainer, behaviorInstruction, this, bindings);
-      controller.created(null);
+      let initialised = false;
+      this.auInit = () => {
```

Comment:
I don't think we need to define it per instance like this to avoid memory pressure. It can be totally a class method. or a loose function. About intialization check, we can use a weakset for it.